Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
   "crates/api",
+  "crates/bootstrap_client",
```

Comment:
```suggestion
  "crates/bootstrap_cli",
```

to match `srv`?

Reply:
With `cli` I'd expect a CLI with a binary.